gpfit/fit.py

```diff
@@ -122,5 +138,35 @@ def evaluate(xdata):
 
     cstrt.evaluate = evaluate
     rms_error = sqrt(mean(square(evaluate(xdata.T)-ydata)))
-
-    return cstrt, rms_error
+    max_error = sqrt(max(square(evaluate(xdata.T)-ydata)))
+
+    def get_dataframe(xdata):
+        " return fit parameters as a dataframe "
+
+        bounds = []
+        if d == 1:
+            bounds.append(min(xdata))
+            bounds.append(max(xdata))
+        else:
+            for i in range(d):
+                bounds.append(min(xdata[i]))
+                bounds.append(max(xdata[i]))
+
+        data = hstack([[ftype, K, d], cs, exs, alpha, exp(bounds),
+                       [rms_error, max_error]])
+        df = pd.DataFrame(data).transpose()
+        colnames = array(["ftype", "K", "d"])
+        colnames = hstack([colnames, ["c%d" % k for k in range(1, K+1)]])
+        colnames = hstack([colnames, ["e%d%d" % (k, i) for k in range(1, K+1)
+                                      for i in range(1, d+1)]])
+        colnames = hstack([colnames, ["a%d" % i for i in
+                                      range(1, len(alpha)+1)]])
+        colnames = hstack([colnames, hstack([["lb%d" % i, "ub%d" % i]
+                                             for i in range(1, d+1)])])
+        colnames = hstack([colnames, ["rms_err", "max_err"]])
+        df.columns = colnames
+        return df
+
+    cstrt.get_dataframe = get_dataframe
```

Comment:
Yay new methods. Now that there's too, may be worth making a `FitConstraintSet` class to hold 'em all in one place?